maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,45 @@
+.technology {
+  align-items: center;
+  background-color: $green;
+  border-radius: $tech-border-radius;
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+  max-width: 100%;
+  padding: 15px;
+  position: relative;
+  width: 200px;
```

Comment:
I think that the width of the cards could be greater because isn't the same the design